client/src/globals.js

```diff
@@ -17,7 +17,8 @@ export const configDefaults = {
   parameters: {
     "max-category-items": 1000,
     "disable-diffexp": false,
-    "diffexp-may-be-slow": false
+    "diffexp-may-be-slow": false,
+    color_config: false
```

Comment:
I can't find any reference to this -- is it dead code?

Reply:
Yes, good catch. Removing this.